From 9f8779eca1560eb5e07f9aa790fb361724540d4d Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 4 Apr 2021 12:35:58 +0300
Subject: [PATCH 1404/1921] arm64: configs: enable cp110 utmi driver in sdk
 builds

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I11d1cdb0a13a6ce3db1493e368f4647edae5d018
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/49154
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
Tested-by: Nadav Haklai <Nadav.Haklai@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index 2e86b3250039..365f227a5e9f 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -781,6 +781,7 @@ CONFIG_PHY_HI6220_USB=y
 CONFIG_PHY_HISTB_COMBPHY=y
 CONFIG_PHY_HISI_INNO_USB2=y
 CONFIG_PHY_MVEBU_CP110_COMPHY=y
+CONFIG_PHY_MVEBU_CP110_UTMI=y
 CONFIG_PHY_QCOM_QMP=m
 CONFIG_PHY_QCOM_QUSB2=m
 CONFIG_PHY_QCOM_USB_HS=y
-- 
2.31.1

